### PR TITLE
Clean up plugin publishing guide

### PR DESCRIPTION
**Pull request by nikhilkh**
@riknoll can you please help review?

### CODE REVIEW

File: www/docs/en/dev/guide/hybrid/plugins/index.md [hunk 1]
```diff
@@ -224,7 +218,21 @@ To publish a plugin to NPM registry you need to follow steps below:
         $ npm adduser # that is if you don't have an account yet
         $ npm publish /path/to/your/plugin
 
-That is it!
+For more details on npm usage refer to [publishing a npm package](https://docs.npmjs.com/getting-started/publishing-npm-packages) on the NPM documentation site.
+
+## Integrating with Cordova Plugin Search
+To surface the plugin in [Cordova Plugin Search](/plugins/), add the `ecosystem:cordova` keyword to the `package.json` file of your plugin and publish it to npm registry.
```
Comment (jcesarmobile) [inline www/docs/en/dev/guide/hybrid/plugins/index.md]: the  "ecosystem:cordova" is added by plugman, right?

File: www/docs/en/dev/guide/hybrid/plugins/index.md [hunk 2]
```diff
@@ -224,7 +218,21 @@ To publish a plugin to NPM registry you need to follow steps below:
         $ npm adduser # that is if you don't have an account yet
         $ npm publish /path/to/your/plugin
 
-That is it!
+For more details on npm usage refer to [publishing a npm package](https://docs.npmjs.com/getting-started/publishing-npm-packages) on the NPM documentation site.
+
+## Integrating with Cordova Plugin Search
+To surface the plugin in [Cordova Plugin Search](/plugins/), add the `ecosystem:cordova` keyword to the `package.json` file of your plugin and publish it to npm registry. 
+
+To indicate support for a particular platform add a keyword with the `<platformName>` as "**cordova-<platformName>**" to the list of keywords in package.json. For example, for a plugin that supports android, iOS & Windows, the keywords in package.json should include:
```
Comment (jcesarmobile) [inline www/docs/en/dev/guide/hybrid/plugins/index.md]: This is added by plugman too. I think this should say that plugman add this important things. It seems that they have to add them manually.

**Comment by nikhilkh** [comment]
@riknoll can you take a quick look as well?

**Comment by riknoll** [comment]
LGTM

File: www/static/plugins/app.js [hunk 3]
```diff
@@ -458,7 +458,7 @@ var App = React.createClass({
                         </div>
                             <div className="plugins_links">
                                     <ul className="nav nav-justified">
-                                    <li><a href="{{ site.baseurl }}/plugins/authors.html"><span className="glyphicon glyphicon-plus"></span><i>&nbsp;</i>Contribute Plugins</a></li>
+                                    <li><a href="{{ site.baseurl }}/docs/en/latest/guide/hybrid/plugins/index.html#publishing-plugins"><span className="glyphicon glyphicon-plus"></span><i>&nbsp;</i>Contribute Plugins</a></li>
```
Comment (dblotsky) [inline www/static/plugins/app.js]: Please use `{{ site.default_linked_docs_version }}` here instead of `/latest/`.

**Comment by dblotsky** [comment]
Why did we decide to stop having this as a page of its own?

**Comment by nikhilkh** [comment]
Plugin publishing guide needed the same content as well - and most of the content was out-of-date.

**Pull request closed without merging**